collated/main/moomeowroar.md

```diff
@@ -0,0 +1,286 @@
+            + COMMAND_WORD + " <new_password> <old_password>";
+
+
+    public static final String MESSAGE_NO_SETTINGS = "No available settings.";
+    public static final String MESSAGE_CHANGE_SUCCESS = "Settings changed successfully.";
+
+    private final String oldPwd;
+    private final String newPwd;
+
+    /**
+     * @param String Name of setting to change
+     */
+    public ChangePwdCommand(String oldPwd, String newPwd) {
+        this.oldPwd = oldPwd;
+        this.newPwd = newPwd;
+    }
+
+    @Override
+    public CommandResult execute() throws CommandException {
+        if (model.getUserPrefs().changePwd(oldPwd, newPwd)) {
+            return new CommandResult(MESSAGE_CHANGE_SUCCESS);
+        }
+        return new CommandResult(MESSAGE_NO_SETTINGS);
+    }
+
+
+}
+```
+###### \java\seedu\address\logic\parser\ChangePwdCommandParser.java
+``` java
+/**
+ * Parses input arguments and creates a new ChangePwdCommand object
+ */
+public class ChangePwdCommandParser implements Parser<ChangePwdCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
+     * and returns an ChangePwdCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ChangePwdCommand parse(String args) throws ParseException {
+        try {
+            String[] array = ParserUtil.parseByDelimiter(args, " ");
+            if (array.length == 2) {
+                return new ChangePwdCommand("", array[1]);
+            } else if (array.length == 3) {
+                return new ChangePwdCommand(array[2], array[1]);
+            } else {
+                throw new ParseException(
+                        String.format(MESSAGE_INVALID_COMMAND_FORMAT, ChangePwdCommand.MESSAGE_USAGE));
+            }
+        } catch (IllegalValueException ive) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ChangePwdCommand.MESSAGE_USAGE));
+        }
+    }
+
+}
+```
+###### \java\seedu\address\logic\parser\ChangeThemeCommandParser.java
+``` java
+/**
+ * Parses input arguments and creates a new ChangePwdCommand object
+ */
+public class ChangeThemeCommandParser implements Parser<ChangeThemeCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
+     * and returns an ChangePwdCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public ChangeThemeCommand parse(String args) throws ParseException {
+        try {
+            String[] array = ParserUtil.parseByDelimiter(args, " ");
+            if (array.length == 2) {
+                if (array[1].equals("light")) {
+                    return new ChangeThemeCommand("LightTheme");
+                } else if (array[1].equals("dark")) {
+                    return new ChangeThemeCommand("DarkTheme");
+                }
```

Comment:
Take note of the magic number used. Can create constants to define what these number means.